ext/Opcode/Opcode.pm

```diff
@@ -417,6 +417,7 @@ These are a hotchpotch of opcodes still waiting to be considered
     once
 
     rv2gv refgen srefgen ref refassign lvref lvrefslice lvavref
+    blessed reftype refaddr
```

Comment:
EXTREME NIT: This is `blessed refaddr reftype` in builtins.pm

Reply:
Fixed f146c8ee14